hooks/post-run.yml

```diff
@@ -1,35 +1,44 @@
 ---
```

Comment:
Change this so
1) We check out Ansible into `job_id/ansible`
2) No need for the tempdir for junit
3) `job_id/junit` & `job_id/junit.xml`
4) Delete dir `job_id`